.internal-ci/README.md

```diff
@@ -70,7 +70,15 @@ v2.0.0-dev
 
 ## CI triggers
 
-This workflow is set up to trigger of certain branch patterns.
+This workflow is set up to trigger off certain branch patterns.
+
+*NOTE:* The branch must contain some change to a source-code file for the CI to be triggered (i.e. branching off `master` to create a feature branch environment without any changes will not trigger CI)
+
+## Deployment Status & Environment Info
```

Comment:
:wrench: As written it makes this heading render as a sibling of `CI triggers` and then places the follow on `Branches` sub heading as a child of this heading.
Suggest moving this before CI Triggers or after the last subheading of CI triggers

Reply:
Ah, good catch...

Reply:
Moved the new section to be a  child of `CI triggers`, and also moved to the end of the list of children of `CI triggers` to not interrupt the flow of the conversation around the branches.  Thanks!